src/test/java/uk/gov/dhsc/htbhf/LocalConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.dhsc.htbhf;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.annotation.PropertySource;
+import uk.gov.dhsc.htbhf.utils.WireMockManager;
+import uk.gov.dhsc.htbhf.utils.WireMockManagerImpl;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+
+@Configuration
+@PropertySource("classpath:application-local.properties")
+@Profile("local")
```

Comment:
Could just have no profile set so that locally no extra parameters are required to run.

Reply:
I'm not sure what you mean here, a local run of the tests needs parameters that a browserstack run doesn't need so I'm not sure how I can do that.

Reply:
We have to have a profile specified so that these beans are not present when we're doing compatibility tests, so a good compromise is to change this so that its `@Profile"(!browserstack)" that means this will be the default if you don't have a profile set and just run it from IntelliJ.